cmd/create/cluster/cmd.go

```diff
@@ -991,6 +991,64 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	cloudAccounts, err := r.OCMClient.GetBillingAccounts()
+	if err != nil {
+		r.Reporter.Errorf("%s", err)
+		os.Exit(1)
+	}
+
+	billingAccounts := ocm.GenerateBillingAccountsList(cloudAccounts)
+
+	if billingAccount == "" && !interactive.Enabled() {
+		// if a billing account is not provided we will try to use the infrastructure account as default
+		if helper.ContainsPrefix(billingAccounts, awsCreator.AccountID) {
+			billingAccount = awsCreator.AccountID
+			r.Reporter.Infof("Using '%s' as billing account", billingAccount)
+			r.Reporter.Infof("To use a different billing account, add --billing-account xxxxxxxxxx to previous command")
+		}
```

Comment:
@chenz4027 I think we have a gap here - what happens if the infrastructure account is not on the list of accounts?

Looks like it will try to create the cluster with an empty billing account.

Reply:
Added error message.